src/less/day-slot.less

```diff
@@ -30,20 +31,25 @@
     min-height: 1em;
   }
 
+  .rbc-time-slot {
+    border-top: 1px solid lighten(@cell-border, 10%);
+  }
 }
 
 .rbc-time-slot {
   flex: 1 0 0;
-  min-height: 20px;
-  height: 20px;
 
-  & + & {
-    border-top: 1px solid @cell-border;
+  &.rbc-now {
+    font-weight: bold;
   }
+}
 
-  &:nth-child(even){
-    border-top-color: lighten(@cell-border, 10%);
-  }
+.rbc-timeslice-group {
+  border-bottom: 1px solid @cell-border;
+  min-height: 40px;
+  height: 40px;
```

Comment:
The entire point of the slices was to shrink them, so that you can fit an entire day onto a single page without scrolling, to allow selecting a large time chunk.  Otherwise you have to change the selection code to scroll the page when the mouse gets to the bottom.  To do the behavior you're describing, all we have to do is change the step value.  The slice (or time slot, as I'm about to rename it) value is about making it tinier.  This is one of the simplybook.me-inspired features.